src/Libraries/RevitNodesUI/Elements.cs

```diff
@@ -190,6 +220,16 @@ public ElementsInView()
             RevitDynamoModel_RevitDocumentChanged(null, null);
         }
 
+        [JsonConstructor]
+        public ElementsInView(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        {
+            DynamoRevitApp.EventHandlerProxy.ViewActivated += RevitDynamoModel_RevitDocumentChanged;
+            DynamoRevitApp.EventHandlerProxy.DocumentOpened += RevitDynamoModel_RevitDocumentChanged;
```

Comment:
same question about disposal here.

Reply:
`ElementsInView` also has it's own `Dispose()` method similar to above